version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-const version = "9"
+const version = "5"
```

Comment:
I think this hurts my soul. Yes … for now?

Reply:
Yes. This is certifiably horrible.